pkg/apis/kubedirector/v1beta1/kubedirectorcluster_types.go

```diff
@@ -182,6 +183,7 @@ type MemberStatus struct {
 	StateDetail      MemberStateDetail `json:"stateDetail,omitempty"`
 	NodeID           int64             `json:"nodeID"`
 	BlockDevicePaths []string          `json:"blockDevicePaths,omitempty"`
+	Affinity         *corev1.Affinity  `json:"affinity,omitempty"`
```

Comment:
What do we gain by copying this into the member status?

Reply:
Good point, we should record the affinity for a given member wrt affinity, right ? Just for book keeping

Reply:
It will always be the same as the role config (since the role config can't be modified while members exist), so I don't think we need to duplicate it here.

It might provide some convenience to have it in the member status, but I think we have to be cautious about expanding the size of the member status since there can be a LOT of these and we have an overall size limit for the CR.

Reply:
Makes sense to me. Removed Affinity from status